libs/acn/PDU.cpp

```diff
@@ -173,6 +186,11 @@ void PDU::PrependFlagsAndLength(ola::io::OutputBufferInterface *output,
     uint8_t flags_and_length[3];
     flags_and_length[0] = static_cast<uint8_t>((size & 0x0f0000) >> 16);
     flags_and_length[0] |= flags;
+    if (force_length_flag) {
+      // TODO(Peter): Should this happen regardless of the force as we're
```

Comment:
same here

Reply:
Likewise.

Reply:
No, no special tests missing, it's just the keyword TODO that caught my eye

Reply:
I've added a note to check these in #1841